src/components/Navbar/SideWidgets/CurrentSpeed.vue

```diff
@@ -1,12 +1,40 @@
 <script setup lang="ts">
 import SpeedCard from '@/components/Core/SpeedCard.vue'
+import SpeedCardClickable from '@/components/Core/SpeedCardClickable.vue'
 import { useMaindataStore, useVueTorrentStore } from '@/stores'
 import { storeToRefs } from 'pinia'
 import { useI18nUtils } from '@/composables'
+import { useTorrentStore } from '@/stores'
+import { TorrentState } from '@/constants/vuetorrent'
+import { computed } from 'vue'
 
 const { t } = useI18nUtils()
 const { serverState } = storeToRefs(useMaindataStore())
 const { displayGraphLimits } = storeToRefs(useVueTorrentStore())
+const { statusFilter } = storeToRefs(useTorrentStore())
+
+const downloadingStates = [TorrentState.DOWNLOADING, TorrentState.DL_FORCED, TorrentState.FORCED_META_DOWNLOAD]
```

Comment:
I'm not sure about the forced meta state, is it really considered as downloading?

Reply:
mostly it means used has clicked on force resume and downloaoding state is metadata downloading. 

I will remove this Just for consistency